docs/versioned/install/README.md

```diff
@@ -8,58 +8,132 @@ function: reference
+| Kubernetes deployment | local, either `kind` or `Minikube` | existing  | existing  |
+| Hardware | 3 CPU, 3 GB RAM | One node:<br>6 CPUs, 6 GB memory, 30 GB disk storage.<br>Multiple nodes:<br>2 CPUs each, 4 GB memory, 20 GB disk storage.   | same as YAML-based |
+
+Supported platforms are Linux, MacOS, and Windows.
 
-Use the following steps to install Knative depending on your installation method.
+The Knative Operator is a custom controller that extends the Kubernetes API to install Knative components. For more information about YAML-based and installations with the Knative Operator, see [YAML and Knative Operator installations compared](#yaml-and-knative-operator-installations-compared).
+
+Use the following steps to install Knative depending on your installation method:
 
 **Quickstart**:
 
   1. Install the [CLI Tools](../client/install-kn.md).
   1. Install the [Knative Quickstart plugin](../getting-started/quickstart-install.md).
 
-**YAML**:
+**YAML-based**:
+
+Install using all YAML files. This option is the most useful if you're using GitOps tools such as [Flux](https://fluxcd.io) or [ArgoCD](https://argo-cd.readthedocs.io/en/stable/) to apply manifests checked into a Git repository. This is the lowest common denominator approach, giving you granular control of the process and resource definitions.
 
   1. Install the [CLI Tools](../client/install-kn.md).
-  1. Install either or both:
-       - Install [Knative Serving](yaml-install/serving/install-serving-with-yaml.md).
-       - Install [Knative Eventing](yaml-install/eventing/install-eventing-with-yaml.md).
+  1. Install either or both components:
+       - Install [Knative Serving](../install/yaml-install/serving/install-serving-with-yaml.md).
+       - Install [Knative Eventing](../install/yaml-install/eventing/install-eventing-with-yaml.md).
 
-**Operator**:
+**Knative Operator**:
+
+Install using the Knative Operator as using manifests or [Helm](https://helm.sh), or install using the Knative Operator that is  installed with the Knative Operator CLI plugin.
 
   1. Install the [CLI Tools](../client/install-kn.md) including the Knative Operator CLI plugin.
-  1. Install using the Knative Operator, and with it the Serving and Eventing components, by either of the following:
-       - The [Knative Operator](./operator/knative-with-operators.md) using K8S Manifests or via Helm.
-       - The [Knative Operator CLI](./operator/knative-with-operator-cli.md).
+  1. Install the Serving and Eventing components, by either of the following:
+       - The [Knative Operator](../install/operator/knative-with-operators.md) using Kubernetes manifests or by using Helm.
+       - The [Knative Operator CLI](../install/operator/knative-with-operator-cli.md).
+
+All installations require a supported Kubernetes version. System requirements provided are recommendations only. The requirements for your installation may vary depending on whether you use optional components, such as a networking layer.
```

Comment:
```suggestion
All installations require a supported Kubernetes version. System requirements provided are recommendations only. The requirements for your installation may vary depending on which plugin components you use.
```

The plugins (like networking or messaging) are not optional in that you need to have at least one installed.  We ship a stand-alone plugin, but you don't have to install the in-memory or Kourier plugins if you have another option installed.